sdk/messaging/azservicebus/internal/amqphandler.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 package internal
 
 //	MIT License
```

Comment:
Remove the license from within the file.  The Copyright statement at the top is sufficient

Reply:
I've removed it from all the other files too, rather than making you point it out each time.